section1/reflection.md

```diff
@@ -1,19 +1,27 @@
+`Strings` = Data type respresented by characters, this can be numbers as well but only as representative characters.
+`Integers/numbers` = Numbers you would use in math and can use similarly. Like 2+2 etc.
+`Floating Point` = Numbers but with decimal. ex. 0.35
 
 3. How would you log the string `"Hello World!"` to the console?
+`console.log("Hello World");`
 
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
-
+You would use `//two back slashes//` or `/* a back slash and an asterik */` Comments are important so that a programmer can look back at your code to understand what you were doing, or how the program works.  It is also helpful to comment out code so you can debug or test your code.
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
+A variable is like a wrapper for a piece of candy.  It will have the name of the candy, and can be referenced by that name and we know what candy the name is talking about.  Wrappers can hold different kinds of candy, but generally once a wrapper holds a piece of candy you don't take it off and use it one a different kind.  
+6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold? I use tumblr quite a bit.  The text posts are probably stored under an individuals username, the text would be of data type string.  On tumblr you can see if you are online or not, I assume this is tracked using a boolean that sees if you have an active browser in Tumblr along with other factors, that then return true or false.  Lastly, you can schedule posts at a certain time, the date may be tracked with a number/integer datatype that takes the time of the scheduled post.
 
-6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
-
-7. In your own words, explain what concatenation is.
+7. In your own words, explain what concatenation is.  
+Concatenation is bringing variables, or local data together, this can be temporary or permanent depending on if you are redefining a variable or just putting out the concat into the console.
 
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
+In facebook they tell you happy birthday, etc.  I am sure they use concatenation to bring your username into the string message.
 
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+Git is going to take time to get used to.  I think there are parts that confuse me about it that I am having a hard time forming into a question.  Mostly I think I am worried about how information is saved.  If I change code but dont commit to git is that  information going to go away?
```

Comment:
If you don't commit it, it is definitely at risk to go away! Once you commit, you can ALWAYS go back to that version. (Well - always, as long as you don't throw computer in the ocean or something - in that case, you would have to had pushed it up to GitHub)